test/htmlparser.spec.js

```diff
@@ -1,36 +1,36 @@
-const expect = require('expect.js')
-
 const HTMLParser = require('../dist/htmlhint.js').HTMLParser
 
-expect.Assertion.prototype.event = function (type, attr) {
-  const obj = this.obj
+expect.extend({
+  toEvent(type, attr) {
+    const obj = this.obj
 
-  if (attr !== undefined) {
-    attr.type = type
-  } else {
-    if (typeof attr === 'string') {
-      attr = { type: type }
+    if (attr !== undefined) {
+      attr.type = type
     } else {
-      attr = type
+      if (typeof attr === 'string') {
+        attr = { type: type }
+      } else {
+        attr = type
+      }
     }
-  }
-  this.assert(
-    eqlEvent(obj, attr),
-    () =>
-      `expected "${JSON.stringify(obj)}" to event "${JSON.stringify(attr)}"`,
-    () =>
-      `expected "${JSON.stringify(obj)}" not to event "${JSON.stringify(attr)}"`
-  )
-}
-
-function eqlEvent(event, attr) {
-  for (const name in attr) {
-    if (name !== 'attrs' && event[name] !== attr[name]) {
-      return false
+    for (const name in attr) {
+      if (name !== 'attrs' && obj && obj[name] !== attr[name]) {
```

Comment:
I had to add the `obj` check here, which makes me think this is just ignoring the problem rather than really fixing this. I don't understand what it's testing enough though